superset/connectors/sqla/views.py

```diff
@@ -167,9 +170,7 @@ class TableModelView(SupersetModelView, DeleteMixin):  # noqa
         'schema': _(
             "Schema, as used only in some databases like Postgres, Redshift "
             "and DB2"),
-        'description': Markup(
-            "Supports <a href='https://daringfireball.net/projects/markdown/'>"
-            "markdown</a>"),
+        'description': _("Supports markdown"),
```

Comment:
First, these descriptions rendered by markdown couldn't be translated.
Second, I thought markdown is a common or easy get language for these who can define the description of Metric.
I can change it back if you still insist on that after you consider above reasons.